### ISSUE TITLE
Misaligned windows after Custom Tile Run

### ISSUE THREAD

**Issue by cogsci2** [giuspen/x-tile#16]
I am going to post a series of images of my preferred dev layout.  It's going to be rather repetitive but each screenshot has a purpose.

1. My setting the layout - layout looks aligned after update
![image](https://user-images.githubusercontent.com/66141096/101325209-3ab54800-3820-11eb-897d-a0ca9486c708.png)
![image](https://user-images.githubusercontent.com/66141096/101325274-56205300-3820-11eb-8b2d-e38065675dc9.png)

2. After ctrl-1 to apply the custom layout, everything is slightly mis-aligned.  a. Slack and Terminal windows occlude the left side of the JupyterLab window.  b) JupyterLab occludes the left side of the Kite window.  c) The Kite window is no longer aligned to the right side of the screen.
![image](https://user-images.githubusercontent.com/66141096/101325374-80721080-3820-11eb-8c5a-90f36b939a86.png)

3. If I pull the left edge of the JupyterLab window to the right, it reveals that the right edges of the Slack and terminal windows aren't aligned, either.  The edge between Slack and Terminal are also not aligned.
![image](https://user-images.githubusercontent.com/66141096/101325471-ac8d9180-3820-11eb-9df8-c240bd08ebeb.png)

Basically, there's overlap on every window.  I'm not sure how to fix this.  Prior to finding x-tile, I have been using custom scripts for the layout.  But I am excited about x-tile as I think it's a far superior solution.
